ast/compile_test.go

```diff
@@ -2577,6 +2592,79 @@ func TestCompilerRewriteExprTerms(t *testing.T) {
 
 			p { __local2__ = [1, 2]; every __local0__, __local1__ in __local2__ { __local1__ } }`,
 		},
+		{
+			note: "every: domain (call)",
+			module: `
+			package test
+
+			p { every x in numbers.range(1, 3) { x } }`,
+			expected: `
+			package test
+
+			p = true {
+				numbers.range(1, 3, __local3__)
+				__local2__ = __local3__
```

Comment:
This is an unfortunate side effect, but it's not a semantic change.

Reply:
Maybe this is an easy condition to control for, though .. 🤔

Reply:
Seems harmless, though. And it's needed for this not to panic, so it's fine 😎